packages/backend/src/server/web/boot.js

```diff
@@ -173,6 +173,7 @@
 			<p>Clear the browser cache / ブラウザのキャッシュをクリアする</p>
 			<p>Update your os and browser / ブラウザおよびOSを最新バージョンに更新する</p>
 			<p>Disable an adblocker / アドブロッカーを無効にする</p>
+	 		<p>&#40;Tor Browser&#41; Set dom.webaudio.enbaled to true / dom.webaudio.enbaledをtrueに設定する</p>
```

Comment:
nit: should be dom.webaudio.en**ab**led, not dom.webaudio.en**ba**led

Reply:
Thanks for the correction